### PR TITLE
[Windows] Sort tool names in README

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
In scope of this PR we are sorting tool names inside each category in alphabetical order.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 1]
```diff
@@ -30,88 +30,91 @@ if (Test-IsWin19)
...
-    (Get-HelmVersion),
-    (Get-ComposerVersion),
-    (Get-NugetVersion),
-    (Get-PipxVersion)
+    (Get-VcpkgVersion),
+    (Get-YarnVersion)
 )
 
 $markdown += New-MDHeader "Project Management" -Level 3
 $markdown += New-MDList -Style Unordered -Lines @(
```
Comment (LeonidLapshin) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:64]: May be we should add ` -Lines (@(...) | Sort-Object)` to every array to be sure that they are sorted properly?

Comment (dibir-magomedsaygitov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Thank you, added `Sort-Object`

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 2]
```diff
@@ -30,111 +30,114 @@ if (Test-IsWin19)
...
-    (Get-GoVersion),
-    (Get-PHPVersion),
     (Get-JuliaVersion),
+    (Get-NodeVersion),
     (Get-PerlVersion),
-    (Get-NodeVersion)
+    (Get-PHPVersion),
+    (Get-PythonVersion),
+    (Get-RubyVersion) | Sort-Object)
 )
```
Comment (maxim-lobanov) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:46]:
```
  (Get-RubyVersion)
  ) | Sort-Object
)
```

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 3]
```diff
@@ -30,111 +30,114 @@ if (Test-IsWin19)
...
-$markdown += New-MDList -Style Unordered -Lines @(
+$markdown += New-MDList -Style Unordered -Lines (@(
     (Get-BrowserVersion -Browser "chrome"),
     (Get-SeleniumWebDriverVersion -Driver "chrome"),
     (Get-BrowserVersion -Browser "edge"),
     (Get-SeleniumWebDriverVersion -Driver "edge"),
     (Get-BrowserVersion -Browser "firefox"),
     (Get-SeleniumWebDriverVersion -Driver "firefox"),
-    (Get-SeleniumWebDriverVersion -Driver "iexplorer")
+    (Get-SeleniumWebDriverVersion -Driver "iexplorer") | Sort-Object)
```
Comment (maxim-lobanov) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:140]: Let's keep order of browsers

File: images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1 [hunk 4]
```diff
@@ -34,38 +34,38 @@ function Get-AndroidInstalledPackages {
...
-            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android SDK Tools"
-        },
         @{
             "Package" = "Android Command Line Tools"
             "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android SDK Command-line Tools"
         },
         @{
-            "Package" = "Android SDK Platforms"
-            "Version" = Get-AndroidPlatformVersions -PackageInfo $packageInfo
+            "Package" = "Android emulator"
```
Comment (al-cheb) [inline images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1:42]: nitpick: "Android emulator" -> ""Android Emulator""

**Review by al-cheb — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

**Review by nikita-bykov — APPROVED** [review]

**Pull request merged**